setup.cfg

```diff
@@ -0,0 +1,4 @@
+[flake8]
```

Comment:
Won't this conflict with black? And why not just use black (the correct version of course)

Reply:
https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length
It should't. Black is a code formatter and flake8 is a linter. After I added this file, my editor doesn't complain about line length.